library/Cube/Web/IdoController.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Icinga\Module\Cube\Web;
 
+use GuzzleHttp\Psr7\ServerRequest;
```

Comment:
Is it possible to use the CompatController as base here? I don't particularly like to have this explicit dependency here.